pkg/control/pubcontrol/pub_control_utils.go

```diff
@@ -175,6 +191,19 @@ func PodUnavailableBudgetValidatePod(client client.Client, control PubControl, p
 	return true, "", nil
 }
 
+// isVirtualKubeletNode Determine whether the node is virtual kubelet
+func isVirtualKubeletNode(cli client.Client, nodeName string) bool {
+	if len(nodeName) == 0 {
+		return false
+	}
+	node := &corev1.Node{}
+	if err := cli.Get(context.Background(), client.ObjectKey{Name: nodeName}, node); err != nil {
+		klog.Errorf("get node(%s) failed: %s", nodeName, err.Error())
+		return false
+	}
+	return node.Labels[labelNodeTypeKey] == virtualNodeType && taints.TaintExists(node.Spec.Taints, taintVirtualToFind)
```

Comment:
I think there is no need to check the taints on node, ` virtualNode` label is enough?

Reply:
done, removed the judgment of taint， `virtualNode` label in enough